versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md

```diff
@@ -585,3 +585,46 @@ You can find a contract example and a comparison with chain extensions
 [here](https://github.com/paritytech/ink/tree/master/integration-tests/call-runtime).
 We've added an example of how to end-to-end test
 `call_runtime` [here](https://github.com/paritytech/ink/tree/master/integration-tests/e2e-call-runtime).
+
+
+### `call_v2`
+
+There is a new host function `call_v2` which allows passing both `Weight` parts: 
+`ref_time_limit` and `proof_time_limit` and the `storage_deposit_limit`. 
+The legacy call function only provides the single `gas_limit` parameter, 
+which is used as the value for `ref_time_limit`.
+
+These parameters can be set on a call builder instance, by first calling `call_v2()` e.g.
+```rust
+call_builder
```

Comment:
do I need to call `call_builder.call_v2`?
or without `call_v2` as in your example?
`call_builder.ref_time_limit(ref_time_limit).proof_time_limit(proof_time_limit)`

Reply:
OK, I misunderstood the description, which suggests calling call_v2() first. It states: 'These parameters can be set on a call builder instance by first calling call_v2().'

Reply:
Fixed